recipe/meta.yaml

```diff
@@ -1,37 +1,39 @@
-{% set version = "4.22" %}
+{% set version = "4.33" %}
 
 package:
   name: nspr
   version: {{ version }}
 
 source:
-  fn: nspr-{{ version }}.tgz
   url: https://ftp.mozilla.org/pub/nspr/releases/v{{ version }}/src/nspr-{{ version }}.tar.gz
-  sha256: c9e4b6cc24856ec93202fe13704b38b38ba219f0f2aeac93090ce2b6c696d430
+  sha256: b23ee315be0e50c2fb1aa374d17f2d2d9146a835b1a79c1918ea15d075a693d7
 
 build:
   number: 0
   skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('nspr', max_pin='x') }}
 
 requirements:
   build:
+    - gnuconfig  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make
     - autoconf
+    - sed
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/libnspr4.a                   # [unix]
     - test -f ${PREFIX}/lib/libnspr4${SHLIB_EXT}         # [unix]
-    - test -f ${PREFIX}/lib/libplc4.a                    # [unix]
     - test -f ${PREFIX}/lib/libplc4${SHLIB_EXT}          # [unix]
-    - test -f ${PREFIX}/lib/libplds4.a                   # [unix]
     - test -f ${PREFIX}/lib/libplds4${SHLIB_EXT}         # [unix]
+    - test -d ${PREFIX}/include/nspr                     # [unix]
+    - test -f ${PREFIX}/bin/nspr-config                  # [unix]
 
 about:
-  home: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR
-  license: MPL 2
+  home: https://hg.mozilla.org/projects/nspr
+  license: MPL-2.0
   license_file: nspr/LICENSE
   summary: A platform-neutral API for system level and libc-like functions.
```

Comment:
We probably will need the `license_family` in the recipe

Reply:
Done